G-Research.OpenTelemetry.Processor.Partial.Tests/G-Research.OpenTelemetry.Processor.Partial.Tests.csproj

```diff
@@ -1,10 +1,9 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net9.0</TargetFramework>
+    <TargetFramework>net8.0</TargetFramework>
```

Comment:
could you try using `<TargetFrameworks>net9.0;net8.0;netstandard2.1;netstandard2.0;net462</TargetFrameworks>` instead? this would align with the upstream OTEL libs (see https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry/OpenTelemetry.csproj#L4 and  https://github.com/open-telemetry/opentelemetry-dotnet/blob/13ed63a88c3a5b42a668bb37d889f15512ccd9cc/build/Common.props#L32)

Reply:
i tried that but got conflicts locally. i'll see to resolve this.

Reply:
done

Reply:
i'm gonna use ones from here since this is going to be part of extensions in the upstream https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/f72971276fc3812d0f0e4c9232fdb22c768066f7/src/OpenTelemetry.Extensions/OpenTelemetry.Extensions.csproj#L5